vnext/ReactUWP/Views/Image/ReactImage.cpp

```diff
@@ -274,7 +274,14 @@ winrt::fire_and_forget ReactImage::SetBackground(bool fireLoadEndEvent) {
           bitmapImage = winrt::BitmapImage{};
 
           strong_this->m_bitmapImageOpened = bitmapImage.ImageOpened(
-              winrt::auto_revoke, [imageBrush](const auto &, const auto &) { imageBrush.Opacity(1); });
+              winrt::auto_revoke, [imageBrush, weak_this, fireLoadEndEvent](const auto &, const auto &) {
+                imageBrush.Opacity(1);
+
+                auto strong_this{weak_this.get()};
+                if (strong_this && fireLoadEndEvent) {
+                  strong_this->m_onLoadEndEvent(*strong_this, true);
```

Comment:
>m_onLoadEndEvent [](start = 31, length = 16)

This code path confuses me.  It runs when we have ImageBrush already set, but this ImageBrush doesn't have a BitmapImage.  @marlenecota what scenario is this code path handling?  If we really do need some extra lagic here, it seems like we also need to listen for ImageFailed.

Reply:
@marlenecota we had an image component that would listen on the image load event and trigger an animation set opacity from 0 to 1. We noticed in some cases when we updated the imageUrl that it didn't fire the load event.

Added ImageFailed now.

Reply:
Thanks!

---
In reply to: [418365658](https://github.com/microsoft/react-native-windows/pull/4750#discussion_r418365658) [](ancestors = 418365658)